### DISCUSSION TITLE
Image is not from a permitted domain

### DISCUSSION THREAD

**Discussion by timetake** [vercel/next.js#25597] [Help]
I'm using Image component on my web site, and it worked very well for the past few week, but starting this morning, the image are not showing , and it give the error "Image is not from a permitted domain" when i clicked the link in console. I'v set it up the domain in next.config.js, with "www.res.cloudinary.com" and "www.datocms-assets.com". I'v searched up, but seems like no one encounter this issue before. im using netify to deploying site, the image showing up correctly on localhost, but not on server.    
```
module.exports = {
  images: {
    domains: [
      'images.unsplash.com',
      'www.res.cloudinary.com',
      'res.cloudinary.com',
      'www.datocms-assets.com',
    ],
  },
  future: {
    webpack5: true,
  },
  target: 'serverless',
};
```
This is how my next.config.js goes

**Comment by bobarros** [comment]
Did you try to remove entries with www?

**Comment by Jaza** [comment]
I don't know if it worked before, because today was the first time I tried using the next.js Image component. But I'm getting exactly the same behaviour. Works fine on localhost, but on Netlify, the `/nextimg` call gives a 403 with response body `Image is not from a permitted domain` .

**Comment by plumreiss** [comment]
Site setting > Environment > Environment variables > edit variables and set key=NEXT_IMAGE_ALLOWED_DOMAINS and value=yourdomain.com.
It should work.
![netlifyenv](https://user-images.githubusercontent.com/74786456/121828288-22216100-cc8d-11eb-80dd-21b84947a8e7.png)

**Comment by Jaza** [comment]
Thanks @plumreiss , that did the trick!

**Comment by knightspore** [comment]
@plumreiss worked for me too! 

Seems this is an ongoing issue under "caveats -> next/image" in [Next on Netlify](https://github.com/netlify/next-on-netlify#nextimage).
